modules/appeals_api/app/swagger/appeals_api/v1/appeals_status.yml

```diff
@@ -0,0 +1,473 @@
+openapi: '3.0.0'
```

Comment:
I copied this yaml file from the equivalent one for v0. I updated the versions and left it the same otherwise. I thought about translating this to rswag, but didn't want to spend the time before I asked - would it help to write some rswag specs to replace this for v1 right now?

Reply:
I'll make a ticket to convert to rswag later. Let's do make this openapi 3.1.0 compatible since our segmented APIs are moving to that version, too.